packages/material-ui/src/Input/Input.d.ts

```diff
@@ -23,6 +23,7 @@ export interface InputProps
   multiline?: boolean;
   name?: string;
   placeholder?: string;
+  readOnly?:boolean;
```

Comment:
You can run `yarn prettier` to fix the code style.

Reply:
done